README.md

```diff
@@ -44,37 +65,38 @@ Plone offers fine-grained permission control over content and actions.
 Plone is easy to set up, extremely flexible,
 and provides you with a system for managing web content that is ideal for project groups, communities, websites, extranets, and intranets.
 
-- *Plone is easy to install.*
+- **Plone is easy to install.**
   Several installation options are available for either your local machine or on servers in the cloud.
 
-- *Plone is easy to use.*
+- **Plone is easy to use.**
   The Plone Team includes usability experts who have made Plone easy and attractive for content managers to add, update, and maintain content.
 
-- *Plone is international.*
+- **Plone is international.**
   The Plone interface has more than 35 translations, and tools exist for managing multilingual content.
 
-- *Plone is standard.*
+- **Plone is standard.**
   Plone carefully follows standards for usability and accessibility.
   Plone pages are compliant with US Section 508, and the W3C's AAA rating for accessibility.
 
-- *Plone is Open Source.*
+- **Plone is Open Source.**
   Plone is licensed under the GNU General Public License, the same license used by Linux.
   This gives you the right to use Plone without a license fee, and to improve upon the product.
 
-- *Plone is supported.*
+- **Plone is supported.**
   There are over three hundred developers in the Plone Development Team around the world, and a multitude of companies that specialize in Plone development and support.
 
-- *Plone is extensible.*
+- **Plone is extensible.**
   There is a multitude of add-on products for Plone to add new features and content types.
   In addition, Plone can be scripted using web standard solutions and Open Source languages.
 
-- *Plone is technology neutral.*
+- **Plone is technology neutral.**
   Plone can interoperate with most relational database systems, open source, and commercial, and runs on a vast array of
   platforms, including Linux, Windows, Mac OS X, Solaris, and BSD.
```

Comment:
I'm not sure about Solaris still being supported. Also maybe combine Linux and BSD? I typically just say "Linux", but I reckon that purists would want to make a distinction.

```suggestion
  platforms, including Linux, Windows, macOS, and BSD.
```

Reply:
I'd like to suggest one extra 'argument' for Plone:

** Plone can't be bought. **

Plone is  Open Source licensend and the code is fully owned by the non profit Plone Foundation to protect the project and code from being bought by any other entitty or competitor to do it any harm like stopping support, development or force you to migrate.